app/src/Controller/ApiController.php

```diff
@@ -31,7 +31,12 @@ public function __construct(GameApi $gameApi, GameRepository $gameRepository, En
   }
 
   public function new(Request $request) {
-    $apiResponse = $this->api->new(self::DEFAULT_SIZE, $request->get('mode') === 'multi');
+    if ($request->getMethod() == 'OPTIONS') {
```

Comment:
I've never seen this kind of code in controller before... Your controller need to be as simple as possible. It must include the minimum logic as possible.